rpcs3/CMakeLists.txt

```diff
@@ -87,6 +87,8 @@ target_sources(rpcs3
     Input/sdl_pad_handler.cpp
     Input/skateboard_pad_handler.cpp
     Input/xinput_pad_handler.cpp
+
+    rpcs3qt/rpcn_countries.cpp
```

Comment:
I just realised that you're in the wrong cmakelists file, lol

Reply:
Which cmake file should this be added to?

Reply:
the one in the rpcs3qt dir of course

Reply:
Ok, will fix accordingly.